homeassistant/components/matter/event.py

```diff
@@ -22,9 +22,20 @@
 from .helpers import get_matter
 from .models import MatterDiscoverySchema
 
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up Matter switches from Config Entry."""
+    matter = get_matter(hass)
+    matter.register_platform_handler(Platform.EVENT, async_add_entities)
+
+
```

Comment:
why did you do this ?
we generally put the constants at the top

Reply:
Since its common to both types of events, I moved it before either type, and grouped constants with related code, i.e., arranged in the form

<common code>
<constants for Generic Switch / followed by code for Generic Switch>
<constants for Lock Events / followed by code for Lock events>

But I didn't know you had the convention of all constants at the top, so if you want to rearrange this to keep that convention, fine with me.

Reply:
Yes, always keep constants at the top of the file